packages/common/src/grpc-service.js

```diff
@@ -161,6 +161,11 @@ function GrpcService(config, options) {
 
   this.grpcMetadata = new grpc.Metadata();
 
+  this.grpcMetadata.add('User-Agent', [
+    config.packageJson.name.replace('@google-cloud', 'gcloud-node'),
```

Comment:
Doesn't it pull in the local <kbd>package.json</kbd> already? Meaning even if we're using the umbrella module, it's sending the individual package version with the request..?